backend/LexBoxApi/Controllers/CrdtController.cs

```diff
@@ -75,7 +75,7 @@ public async Task<ActionResult<int>> CountChanges(Guid projectId,
     }
 
     [HttpGet("listProjects")]
-    public async Task<ActionResult<FieldWorksLiteProject[]>> ListProjects()
+    public async Task<ActionResult<ListProjectsResult>> ListProjects()
```

Comment:
I'd like to avoid breaking any existing clients, can we make a new API which returns the new ListProjectsResult and preserve this API. Not sure what a good request path would be. Maybe `GET /api/crdt/projectAccess`? we can always remove `listProjects` in a month or so when it's reasonable to expect people to have upgraded.

Reply:
Oh right, we have users out there now. When we discussed it in the office I was blithe about breaking APIs without needing to version because we're the only ones using this API, but we can't be quite so cavalier about it because we do have older versions of our client out there. Right; I'll undo that change and create a new FW Lite API endpoint.

Reply:
I went with `listProjectsV2`, with the intention of switching it to `listProjects` once clients have upgraded and then retiring `listProjectsV2` later after another round of client upgrades. (Which should happen fairly often as we're still very much in beta).

Done in commit 4646946a9.